roles/pulp_common/handlers/main.yml

```diff
@@ -11,15 +11,19 @@
 # begin with.
 - name: Restore SELinux contexts on Pulp dirs that must exist
   command: '/sbin/fixfiles restore {{ __pulp_selinux_label_dirs | join(" ") }}'
-  # We need the when condition on this particular SELinux handler because it
-  # gets triggered by "Collect static content".
+  when:
+    - ansible_facts.os_family == 'RedHat'
+    - ansible_facts.selinux.status == "enabled"
```

Comment:
Do we need `ansible_facts.os_family == 'RedHat'`?

Reply:
It is a  copy/paste of https://github.com/pulp/pulp_installer/blob/master/roles/pulp_common/handlers/main.yml#L44-L47 few lines below. Mainly to remain consistent. If you want me to remove it I am fine (I suppose on non-Red Hat family selinux.status won't return enabled)

Reply:
Well we don't test that, but i won't be so sure:
https://wiki.debian.org/SELinux
https://wiki.ubuntu.com/SELinux
https://wiki.archlinux.org/index.php/SELinux
just to name a few...

Reply:
That said, maybe they need different commands and that guard should be in place.

Are Fedora and CentOS falling into `os_family == "RedHat"`?

Reply:
For the latter question yes. And I didn't know they all ported SELinux already. Mea culpa.

Reply:
No problem, let's keep it that way until there is demand.
Still, someone with more SELinux insight should take a look.

Reply:
Yeah, I have 2 reasons for this:
1. We only install the package needed for the variable `ansible_facts.selinux` to exist on the RedHat family. It gets installed via the per-distro variable `pulp_preq_packages`, in the task `Install prerequisites`.

2. Until someone does the (testing) work for Pulp 3's SELinux support on those distros, I'd rather not have any SELinux codepaths for them in pulp_installer, nor install a package for it. This is short-circuit evaluation (when conditions as a list, rather than with `and`).

Reply:
And @mdellweg yes, Fedora and CentOS do indeed fall under os_family == "RedHat".